files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,442 @@ class AaaCfg(object):
+        cmd = 'ip link add lo-m type dummy'
+        if run_command(cmd) == False:
+            return
+
+        cmd = 'ip addr add 127.0.0.1/8 dev lo-m'
+        if run_command(cmd) == False:
+            return
+
+        cmd = 'ip link set lo-m up'
+        if run_command(cmd) == False:
+            return
+
+        cmd = 'ip link set dev lo-m master mgmt'
+        if run_command(cmd) == False:
+            return
+
+        self.bind_mgmt_intf()
+
+    def bind_mgmt_intf(self):
+        syslog.syslog(syslog.LOG_INFO, "Handle MGMT intf:")
+        keys = self.config_db.get_keys(CFG_MGMT_INTF_TABLE)
+        syslog.syslog(syslog.LOG_INFO, "MGMT interfaces keys: {}".format(str(keys)))
+
+        mgmt_intf_bound = False
+        for it in keys:
+            if type(it) is unicode:
+                mgmt_intf_name = it
+
+                cmd = 'ip link set dev {} master mgmt'.format(mgmt_intf_name)
+                if run_command(cmd) == False:
+                    return
+
+                cmd = 'sysctl -w net.ipv4.tcp_l3mdev_accept=1'
+                if run_command(cmd) == False:
+                    return
+
+                mgmt_intf_bound = True
+                break
+        if not mgmt_intf_bound:
+            syslog.syslog(syslog.LOG_ERR, "MGMT VRF: MGMT interface not bound")
+        else:
+            for it in keys:
+                if type(it) is unicode:
+                    key = it
+                else:
+                    key = it[0] + "|" + it[1]
+
+                entry = self.config_db.get_entry(CFG_MGMT_INTF_TABLE, key)
+                if 'gwaddr' in entry:
+                    mgmt_intf_gwaddr = entry['gwaddr']
+                    cmd = 'ip route add table {} 0.0.0.0/0 via {}'.format(MGMT_VRF_TABLE_ID, mgmt_intf_gwaddr)
+                    if run_command(cmd) == False:
+                        return
+
+    def unconfigure_mgmt_vrf(self):
+        syslog.syslog(syslog.LOG_INFO, "Unconfigure MGMT VRF:")
+
+        cmd = 'ip link delete lo-m'
+        if run_command(cmd) == True:
+            self.delete_mgmt_vrf_table()
+
+        return
+
+    def handle_mgmt_intf_gwaddr_cfg(self, op, ifname, gwaddr):
+        syslog.syslog(syslog.LOG_INFO, "Received MGMT Intf gwaddr config, op: {}, ifname: {}, gwaddr: {}".format(op, ifname, gwaddr))
+        keys = self.config_db.get_keys(CFG_MGMT_VRF)
+        if keys:
+            cmd = 'ip route del table {} 0.0.0.0/0'.format(MGMT_VRF_TABLE_ID)
+            run_command(cmd)
+            
+            if op == "ADD":
+                cmd = 'ip route add table {} 0.0.0.0/0 via {}'.format(MGMT_VRF_TABLE_ID, gwaddr)
+                run_command(cmd)
+            return True
+
+    def handle_mgmt_intf_creation(self, mgmt_intf_key):
+        syslog.syslog(syslog.LOG_INFO, "Received MGMT Intf creation, key: {}".format(mgmt_intf_key))
+        keys = self.config_db.get_keys(CFG_MGMT_VRF)
+        if keys:
+            syslog.syslog(syslog.LOG_INFO, "MGMT VRF created")
```

Comment:
Please remove all the unwanted logs. Looks like you've added these for debugging purpose. Also provide data in logs instead of just statements

Reply:
addressed